opencog/python/web/api/apimain.py

```diff
@@ -61,6 +65,16 @@ def __init__(self, atomspace):
         self.api.add_resource(scheme_api,
                               '/api/v1.1/scheme',
                               endpoint='scheme')
+        self.api.add_resource(ghost_api, 
+                              '/api/v1.1/ghost', 
+                              endpoint='ghost')
+
+
+    def allow_cross_domain(self, response):
+        """Hook to set up response headers."""
+        response.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
I'm unclear as to what this is about ... cross-domain scripting is considered to be one of the great evils of the last decade, its that infamous technique that allows systems to be compromised, hijacked. 

Now, I don't really understand quite what this is going to do, so perhaps its totally harmless. Still, turning off all access control is traditionally one of those dangerous things to do... can this be double-checked?  What happens if these lines of code are removed?

Reply:
This is a hack that I created because the pre-flight request was pending when I tried to connect to the scheme endpoint. It might show up on your local machine if you remove this and try to send queries to the scheme end-point. But we don't need this for the ghost end-point so you can go head and remove it. And mike took the code locally from us when he was in addis that is why such hacks where in the pull-request. Remove the whole funciton definition and also it usuage at line 90.